### PR TITLE
updates to reflect changes to validation summary view

### PR DESCRIPTION
**Pull request by dgrossen**
## Description

Updates to reflect changes made to the Assurance Tools Validation Summary Plugin View

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/platform-assurance/tutorials/validation-summary.md [hunk 1]
```diff
@@ -46,34 +41,33 @@ After you have tested your function, you can select **Upload** to save your vali
...
 | `events` | Array | An array of event uuids to be reported as matched or not matched. |
+| `links` | Array | An array of ValidationResultLink objects {( type: 'doc'|'product', url: String )} to reference documentation and other resources |
 | `result` | String | This is the validation result and is expected to be one of the enumerated strings: "matched", "not matched", "unknown" |
 
-![](./assets/validation-summary/invalid.png)
-
 ## Viewing Results
 
-The left panel displays the results of all enabled validations, including the tested function in the right panel.
+The results of the function will be displayed in the results section just below the code editor. When a validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, then those events will be highlighted in the timeline with the following colors:
```
Comment (hbhandal80) [inline src/pages/documentation/platform-assurance/tutorials/validation-summary.md:60]: Let's rephrase this slightly: "The results of the function are displayed in the results section below the code editor. If the validation result is `unknown` or `not matched` and the `events` array has one or more `uuids`, the events will be highlighted in the timeline with the following colors:"

File: src/pages/documentation/platform-assurance/tutorials/validation-summary.md [hunk 2]
```diff
@@ -46,34 +41,33 @@ After you have tested your function, you can select **Upload** to save your vali
...
-
-Additionally, you can also reach out to our team on our Slack Channel at [https://projectgriffon.slack.com/](https://projectgriffon.slack.com/).
+You can add `console.log()` in your function to print items to the developer console. Alternatively, you can use the message property of the result object to debug messages to the results panel.
 
-To learn more about validations, you can visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/assurance-validation-plugins) GitHub. There you can find examples from validations owned by Adobe, as well as the [wiki](https://github.com/adobe/assurance-validation-plugins/wiki) that describes validations in more detail.
+If an error occurs in the JavaScript code editor, an error status is displayed along with the reason.
 
+To learn more about validations, you can visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/griffon-validation-plugins) GitHub. There you can find examples from validations owned by Adobe, as well as the [wiki](https://github.com/adobe/griffon-validation-plugins/wiki) that describes validations in more detail.
```
Comment (hbhandal80) [inline src/pages/documentation/platform-assurance/tutorials/validation-summary.md:73]: Rephrase the last paragraph to: "To learn more about validations, please visit the [Adobe Experience Platform Assurance Validations](https://github.com/adobe/griffon-validation-plugins) GitHub, where you will find examples of validations owned by Adobe. See the [wiki](https://github.com/adobe/griffon-validation-plugins/wiki) for more detailed descriptions of validations."

**Review by hbhandal80 — CHANGES_REQUESTED** [review]
A few comments.

**Review by hbhandal80 — APPROVED** [review]
Thanks for the updates. Looks good to me.

File: src/pages/documentation/platform-assurance/tutorials/validation-summary.md [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 
-You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in an Adobe Experience Platform Assurance session. Each function receives the events in an Assurance session. You can write functions to validate your client configuration, event conditions, tests and use cases.
+You may use the Validation Editor feature to quickly and easily manage JavaScript functions to validate events in an Adobe Experience Platform Assurance session. Each function receives the events in an Assurance session. You can write functions to validate your client configuration, event conditions, tests and use cases.
 
 ## Getting Started
 
 To use this view, complete the following steps:
 
-1. [Set up Adobe Experience Platform Assurance](../set-up.md)
-2. [Create](../index.md#creating-sessions) and [connect](../index.md#connecting-to-a-session) to a Assurance session
```
Comment (emdobrin) [inline src/pages/documentation/platform-assurance/tutorials/validation-summary.md:12]: What was the reason for this change, the old links seem to be working - https://aep-sdks.gitbook.io/docs/adobe-assurance/project-griffon/using-project-griffon/validation-summary-and-project-griffon#getting-started

File: src/pages/documentation/platform-assurance/tutorials/custom-validation.md [hunk 4]
```diff
@@ -1,68 +0,0 @@
-# Custom Validation
```
Comment (emdobrin) [inline src/pages/documentation/platform-assurance/tutorials/custom-validation.md:1]: is it safe to remove this altogether or should we have a deprecation warning on top going for a while?

File: src/pages/documentation/platform-assurance/tutorials/validation-summary.md [hunk 5]
```diff
@@ -36,7 +31,7 @@ After you have tested your function, you can select **Upload** to save your vali
 | `uuid` | String | Universally unique identifier for the event. |
 | `timestamp` | Number | Timestamp from the client when the event was sent to Assurance. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format (for example, com.adobe.assurance). |
```
Comment (emdobrin) [inline src/pages/documentation/platform-assurance/tutorials/validation-summary.md:39]: formatting and renaming - can we revert this change to remove the backslashes and use the new name?

File: src/pages/documentation/platform-assurance/tutorials/custom-validation.md (hunk 4 above)
Comment (hbhandal80) [reply src/pages/documentation/platform-assurance/tutorials/custom-validation.md:1]: Agree with Emilia - It's good practice to have a deprecation warning to provide awareness to customers.

**Review by hbhandal80 — COMMENTED** [review]
Just the 1 comment regarding deprecation.

File: src/pages/documentation/platform-assurance/tutorials/custom-validation.md (hunk 4 above)
Comment (dgrossen) [reply src/pages/documentation/platform-assurance/tutorials/custom-validation.md:1]: this assurance view was deprecated a few years ago and has been removed from the UI for well over a year. I don't think it does any good to leave this page up.

Comment (emdobrin) [reply src/pages/documentation/platform-assurance/tutorials/custom-validation.md:1]: 👍 and thanks for the clarification

**Review by emdobrin — APPROVED** [review]
